var/spack/repos/builtin/packages/py-h5glance/package.py

```diff
@@ -16,6 +16,8 @@ class PyH5glance(PythonPackage):
     url      = "https://pypi.io/packages/source/h/h5glance/h5glance-0.4.tar.gz"
 
     version('0.4', sha256='03babaee0d481991062842796126bc9e6b11e2e6e7daba57c26f2b58bf3bbd32')
+    version('0.5', sha256='bc34ee42429f0440b329083e3f67fbf3d7016a4aed9e8b30911e5905217bc8d9')
+    version('0.6', sha256='203369ab614273aaad3419f151e234609bb8390b201b65f678d7e17c57633e35')
```

Comment:
Can you sort these from newest to oldest?

Reply:
Done :slightly_smiling_face: